utils/backport.py

```diff
@@ -27,12 +28,13 @@
 print(f'Backporting {len(commits)} commits from "{title}"')
 branch = f"backport-{args.pr}"
 base = f"release/{args.version}"
-subprocess.check_call(["git", "fetch", "origin"])
-subprocess.check_call(["git", "checkout", "-b", branch, f"origin/{base}"])
+remote = args.remote
+subprocess.check_call(["git", "fetch", f"{remote}"])
```

Comment:
No need to use a f-string when the variable is the whole string, just use it directly :)

(tbh I'm surprised ruff didn't flag it)

Reply:
:heavy_check_mark: